common/include/Utilities/IniInterface.h

```diff
@@ -70,10 +70,11 @@ class IniInterface
 	void EnumEntry( const wxString& var, T& value, const wxChar* const* enumArray=NULL, const T defvalue=(T)0 )
 	{
 		int tstore = (int)value;
+		int defaultvalue = (int)defvalue;
```

Comment:
why? To support the enum class? But why did you convert cdvd source to an enum class, actually ?